### ISSUE TITLE
需要learning rate decay/ schedule的文档

### ISSUE THREAD

**Issue by jmliu88** [PaddlePaddle/Paddle#4295]
在目前的版本里没找到关于learning rate decay或者learning rate schedule用法的文档，可以麻烦哪位同学补一下么？

谢谢

**Comment by lcy-seso** [comment]
学习率退火的参数含义见此issue下的回复：
https://github.com/PaddlePaddle/Paddle/issues/1167#issuecomment-273373637

设置方法可以参考此例：
https://github.com/PaddlePaddle/models/pull/220/files#diff-0c748d8eb579074fe17993b39117a805R206

我们会尽快修正文档。

**Comment by ranqiu92** [comment]
文档已更新，可以参考 https://github.com/PaddlePaddle/Paddle/blob/develop/doc/faq/index_cn.rst#learning-rate-annealing

**Comment by ranqiu92** [comment]
文档已更新，可以参考https://github.com/PaddlePaddle/Paddle/blob/develop/doc/faq/index_cn.rst#learning-rate-annealing

**Comment by sshilei** [comment]
接着问一下，如何在SGD中使用 learning rate decay

**Comment by sshilei** [comment]
sgd_optimizer = fluid.optimizer.SGD(learning_rate=0.2)

**Comment by qingqing01** [comment]
@sshilei 

- 在Paddle Fluid里可以各类学习率调整策参考文档:  http://paddlepaddle.org/documentation/docs/zh/1.4/api_cn/layers_cn.html#permalink-234-learning_rate_scheduler

- 具体使用可参考： http://paddlepaddle.org/documentation/docs/zh/1.4/api_cn/layers_cn.html#exponential-decay

**Comment by sshilei** [comment]
api里面的参数是按step进行下降，有没有接口可以根据epoch进行下降

**Comment by sandyhouse** [comment]
试试下面这个：(http://paddlepaddle.org/documentation/docs/zh/1.4/api_cn/layers_cn.html#permalink-234-learning_rate_scheduler)

paddle.fluid.layers.cosine_decay(learning_rate, step_each_epoch, epochs)

说明：使用 cosine decay 的衰减方式进行学习率调整。
在训练模型时，建议一边进行训练一边降低学习率。 通过使用此方法，学习速率将通过如下cosine衰减策略进行衰减：
decayed_lr=learning_rate∗0.5∗(cos(epoch∗math.pi/epochs)+1)

参数：

- learning_rate （Variable | float） - 初始学习率。

- step_each_epoch （int） - 一次迭代中的步数。

- epochs - 总迭代次数。


代码示例：
base_lr = 0.1
lr = fluid.layers.cosine_decay( learning_rate = base_lr, step_each_epoch=10000, epochs=120)
